// slack/methods/channels.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _channels = require('./channels.archive');

var _channels2 = _interopRequireDefault(_channels);

var _channels3 = require('./channels.create');

var _channels4 = _interopRequireDefault(_channels3);

var _channels5 = require('./channels.history');

var _channels6 = _interopRequireDefault(_channels5);

var _channels7 = require('./channels.info');

var _channels8 = _interopRequireDefault(_channels7);

var _channels9 = require('./channels.invite');

var _channels10 = _interopRequireDefault(_channels9);

var _channels11 = require('./channels.join');

var _channels12 = _interopRequireDefault(_channels11);

var _channels13 = require('./channels.kick');

var _channels14 = _interopRequireDefault(_channels13);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
  archive: _channels2.default,
  create: _channels4.default,
  history: _channels6.default,
  info: _channels8.default,
  invite: _channels10.default,
  join: _channels12.default,
  kick: _channels14.default,
  leave: _channels14.default,
  list: _channels14.default,
  mark: _channels14.default,
  rename: _channels14.default,
  setPurpose: _channels14.default,
  setTopic: _channels14.default,
  unarchive: _channels14.default
};
module.exports = exports['default'];
